docs/known-issues.md

```diff
@@ -9,3 +9,17 @@ This is due to design limitation in TypeScript that augmentation does not work w
 <https://github.com/Microsoft/TypeScript/issues/9532>
 
 Workaround: TBD
+
+## Global Dependencies
+
+If the typings has global dependencies listed, it will not be installed with the typings automatically.
+The consumer will see a `INFO` message like `"\<typings x\>" lists global dependencies on "\<typings y\>" and should be installed`.
+Consumer needs to install them manually.
+
+This is a limitation because global dependencies, by definition, are global. They declare themselves in the [global environment](http://www.ecma-international.org/ecma-262/7.0/index.html#sec-lexical-environments).
+
+Because of that, if there are two modules depends on the same global dependency, that dependency will be installed twice and cause "Duplicate Identifier" errors.
+
+Workaround:
+- Create typings in external module format (aka module file with ambient module declaration), or
```

Comment:
Right, it's all good. I think I'd just avoid the parens altogether myself - isn't it just expressing the same information? Because of Typings, the word ambient is overloaded in the context of TypeScript and probably best avoided - the word typings or definition implies it's ambient (otherwise it'd be a TypeScript file and not a definition/typing).